### PR TITLE
Restructure Readme

### PR DESCRIPTION
**Pull request by Bibo-Joshi**
Restructures the readme a bit to more prominently point to our documentation & wiki. As a follow-up of this PR, I'll also adjust rules-bot to offer a taghint that points to the `working with ptb` section. That way we'll have a better starting point for newbies

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -150,14 +168,23 @@ Additionally, two shortcuts are provided:
...
+Once you have installed, the library, you can begin working with it, so let's get started!
+
 Quick Start
-===========
+~~~~~~~~~~~
 
 Our Wiki contains an `Introduction to the API <https://github.com/python-telegram-bot/python-telegram-bot/wiki/Introduction-to-the-API>`_ explaining how the pure Bot API can be accessed via ``python-telegram-bot``.
 Moreover, the `Tutorial: Your first Bot <https://github.com/python-telegram-bot/python-telegram-bot/wiki/Extensions---Your-first-Bot>`_ gives an introduction on how chatbots can be easily programmed with the help of the ``telegram.ext`` module.
 
+For additional resources, please see below.
```
[the remark this answers is not in the document]
Comment (Bibo-Joshi) [reply README.rst:184]: is it really so important? 🤔 I'm okay with skipping though

File: README.rst [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+    <div style="display: none">
+
+Hidden Headline
+===============
+This is just here to get furo to display the right sidebar.
+
+.. raw:: html
+
+    </div>
+
```
[the remark this answers is not in the document]
Comment (Bibo-Joshi) [reply README.rst:12]: huh, thanks for the catch!

File: README.rst [hunk 3]
```diff
@@ -76,13 +76,18 @@ In addition to the pure API implementation, this library features a number of hi
 make the development of bots easy and straightforward. These classes are contained in the
```
Comment (Poolitzer) [inline README.rst:76]: maybe we should also mention all our shortcuts here? e.g.

_In addition to the pure API implementation, this library features convenient shortcuts to save you from re-passing variables and_

this comment should go a line higher, github doesnt let me.

File: README.rst [hunk 4]
```diff
@@ -76,13 +76,18 @@ In addition to the pure API implementation, this library features a number of hi
...
 ``telegram.ext`` submodule.
 
+After installing_ the library, be sure to check out the section on `working with PTB`_.
+
 Telegram API support
-====================
+~~~~~~~~~~~~~~~~~~~~
+
+All types and methods of the Telegram Bot API **7.7** are natively supported by this library.
+In addition, Bot API functionality not yet natively included can still be used as described `in our wiki <https://github.com/python-telegram-bot/python-telegram-bot/wiki/Bot-API-Forward-Compatibility>`_.
```
Comment (Poolitzer) [inline README.rst:85]:
```suggestion
In addition, Bot API functionality not yet natively included can still be achieved via methods described `in our wiki <https://github.com/python-telegram-bot/python-telegram-bot/wiki/Bot-API-Forward-Compatibility>`_.
```

File: README.rst [hunk 5]
```diff
@@ -76,13 +76,18 @@ In addition to the pure API implementation, this library features a number of hi
...
 
-All types and methods of the Telegram Bot API **7.7** are supported.
+.. _installing:
```
Comment (Poolitzer) [inline README.rst:87]: is that needed for linking? should just work with headers no

oh I guess for furo? Can you comment this in if that is the case?

**Review by Poolitzer — CHANGES_REQUESTED** [review]

File: README.rst (hunk 4 above)
Comment (Bibo-Joshi) [reply README.rst:85]: "bot API functionality […] can be achieved via methods" sounds wrong to me …

File: README.rst (hunk 3 above)
Comment (Bibo-Joshi) [reply README.rst:76]: nice idea. Tried to tie it in in the same sentence :)

File: README.rst (hunk 5 above)
Comment (Bibo-Joshi) [reply README.rst:87]: you're right, it works without :)

File: README.rst (hunk 4 above)
Comment (Bibo-Joshi) [reply README.rst:85]: I'll still not merge that ;P "achieving bot api functionality" sounds like we're implementing the api server rather than using it.

**Comment by harshil21** [comment]
Also expanding on my idea for the readme (https://t.me/c/1494805131/14589), I'd like to add a "Features" section. Here is one generated by an LLM:

> Features:
> * Fully asynchronous ⚡
> * Convenient shortcut methods 🚀
> * Type hinting support for easy development 🔍
> * Customizable interface 🛠️
> * Seamless integration with webhooks and polling 🔄
> * Comprehensive documentation and examples 📚

**Comment by Poolitzer** [comment]
@harshil21 Personally not a fan of emojis, I prefer a bullet point list.

I don&#x27;t have anything against this summarising in general though.

**Comment by Bibo-Joshi** [comment]
I'm with pool on the emojis here 😅 But I added a section and also included corresponding links :)

**Review by harshil21 — APPROVED** [review]

**Review by Poolitzer — APPROVED** [review]
LGTM

**Pull request merged**